packages/fx-core/src/component/driver/aad/utility/aadAppClient.ts

```diff
@@ -99,6 +100,53 @@ export class AadAppClient {
     });
   }
 
+  public async getOwners(objectId: string): Promise<AadOwner[] | undefined> {
+    // const response = await instance.get(`${baseUrl}/applications/${objectId}/owners`);
+    const response = await this.axios.get(`applications/${objectId}/owners`, {
+      "axios-retry": {
+        retries: this.retryNumber,
+        retryDelay: axiosRetry.exponentialDelay,
+        retryCondition: (error) =>
+          axiosRetry.isNetworkError(error) ||
+          axiosRetry.isRetryableError(error) ||
+          this.is404Error(error), // also retry 404 error since AAD need sometime to sync created AAD app data
+      },
+    });
+
+    const aadOwners: AadOwner[] = [];
+    if (response && response.data && response.data.value) {
+      for (const aadOwner of response.data.value) {
+        aadOwners.push({
+          userObjectId: aadOwner.id,
+          resourceId: objectId,
+          displayName: aadOwner.displayName,
+          // For guest account, aadOwner.userPrincipalName will contains "EXT", thus use mail instead.
+          userPrincipalName: aadOwner.mail ?? aadOwner.userPrincipalName,
+        });
+      }
+      return aadOwners;
+    }
+
+    return undefined;
+  }
+
+  public async addOwner(objectId: string, userObjectId: string): Promise<void> {
+    const requestBody = {
+      "@odata.id": `${this.axios.defaults.baseURL}/directoryObjects/${userObjectId}`,
+    };
+
+    await this.axios.post(`applications/${objectId}/owners/$ref`, requestBody, {
+      "axios-retry": {
```

Comment:
can also remove the last parameter (axios-retry config) if you don't need to overwrite the default retry policy. See L61 for example and L44 for the default retry policy.